eradiate/scenes/atmosphere/_homogeneous.py

```diff
@@ -16,11 +24,42 @@ class HomogeneousAtmosphere(Atmosphere):
     """
     Homogeneous atmosphere scene element [:factorykey:`homogeneous`].
 
-    This class builds an atmosphere consisting of a homogeneous medium.
-    Scattering uses the Rayleigh phase function.
+    This class builds an atmosphere consisting of a homogeneous medium with
+    customisable collision coefficients and phase function, attached to a
+    cuboid shape.
     """
 
-    sigma_s = documented(
+    _bottom: pint.Quantity = documented(
+        pinttr.ib(
+            default=ureg.Quantity(0.0, ureg.km),
+            units=ucc.deferred("length"),
+            converter=pinttr.converters.to_units(ucc.deferred("length")),
+            validator=pinttr.validators.has_compatible_units,
```

Comment:
Not required (having this handled by default is the whole point of Pinttrs ;) ).